src/app/app.component.html

```diff
@@ -1,49 +1,126 @@
-<div class="wrapper  w-full flex justify-center">
+<div class="wrapper w-full flex justify-center">
   <div class="w-full max-w-[1440px]">
-    <nav class="bg-transparent w-full z-20 text-black flex justify-between items-center h-[90px] max-w-[1200px] mx-auto mb-[57px]">
-      <div class="h-[70px] justify-between items-center w-full flex">
+    <nav
+      class="w-full bg-[#FFFFFF] fixed md:static z-20 text-black flex justify-between items-center h-[90px] max-w-[1200px] mx-auto lg:mb-[57px] md:mb-0 px-5 md:px-0">
```

Comment:
The heade on mobile device so hight, please use the classes:

```
w-full bg-[#FFFFFF] fixed md:static z-20 text-black flex justify-between items-center h-[60px] md:h-[90px] max-w-[1200px] mx-auto lg:mb-[57px] md:mb-0 px-5 md:px-0
```

Reply:
done